src/content/docs/fr/develop-and-build.mdx

```diff
@@ -0,0 +1,146 @@
+
+Chaque modèle de démarrage contient un script préconfiguré qui démarrera `astro dev` pour vous. Après avoir navigué dans le dossier de votre projet, utilisez votre gestionnaire de paquets préféré pour exécuter cette commande et démarrer le serveur de développement Astro.
+
+<Tabs>
+  <TabItem label="npm">
+  ```shell
+  npm run dev
+  ```
+  </TabItem>
+  <TabItem label="pnpm">
+  ```shell
+  pnpm run dev
+  ```
+  </TabItem>
+  <TabItem label="yarn">
+  ```shell
+  yarn run dev
+  ```
+  </TabItem>
+</Tabs>
+
+
+Si tout se passe bien, Astro va désormais servir votre projet sur [http://localhost:4321/](http://localhost:4321/). Visitez cette adresse dans votre navigateur et voir votre nouveau site !
+
+### Travailler en mode développement
+
+Astro va écouter les modifications en direct dans votre répertoire `src/` et mettre à jour votre aperçu du site en le construisant, de sorte que vous n'aurez pas à redémarrer le serveur à chaque fois que vous apportez des modifications pendant le développement. Vous serez toujours en mesure de voir une version à jour de votre site dans votre navigateur lorsque le serveur de développement est en cours d'exécution.
+
+Lorsque vous consultez votre site dans votre navigateur, vous aurez accès à la [barre d'outils de développement d'Astro](/fr/guides/dev-toolbar/). Au fur et à mesure que vous construisez, elle vous aidera à inspecter vos [îles](/fr/concepts/islands/), à repérer les problèmes d'accessibilité et plus encore.
+
+Si vous ne pouvez pas ouvrir votre projet dans votre navigateur après avoir démarré le serveur de développement, revenez à la console où vous avez lancé la commande `dev` et vérifiez le message affiché. Il devrait vous indiquer si une erreur s'est produite ou si votre projet est servi à l'URL différente de [http://localhost:4321/](http://localhost:4321/).
+
+## Construire et prévisualiser votre site
+
+Pour vérifier la version de votre site qui sera créée à l'exécution du build, quittez le serveur de développement (<kbd>Ctrl</kbd> + <kbd>C</kbd>) et exécutez la commande de construction appropriée pour votre gestionnaire de paquets dans votre terminal :
+
+    <Tabs>
+      <TabItem label="npm">
+      ```shell
+      npm run build
+      ```
+      </TabItem>
+      <TabItem label="pnpm">
+      ```shell
+      pnpm build
+      ```
+      </TabItem>
+      <TabItem label="yarn">
+      ```shell
+      yarn run build
+      ```
+      </TabItem>
+    </Tabs>
+
+Astro va construire une version prête à déployer de votre site dans un dossier séparé (`dist/` par défaut) et vous pouvez surveiller son progrès dans la console. Cela vous avertira des erreurs de construction dans votre projet avant de déployer sur le web. Si TypeScript est configuré pour `strict` ou `strictest`, la commande `build` de `build` vérifiera également votre projet pour les erreurs de type.
```

Comment:
nit: I don't know if this just me by I'd say there is a slight difference in meaning between the two
```suggestion
Astro va construire une version prête à déployer de votre site dans un dossier séparé (`dist/` par défaut) et vous pouvez surveiller sa progression dans la console. Cela vous avertira des erreurs de construction dans votre projet avant de déployer sur le web. Si TypeScript est configuré pour `strict` ou `strictest`, la commande `build` de `build` vérifiera également votre projet pour les erreurs de type.
```